src/lib/storage.test.ts

```diff
@@ -15,21 +97,22 @@ describe("Storage", () => {
   it("should set an item", () => {
     const mockSender = jest.fn();
     storage.registerSender(mockSender);
-
+    const key = "key1";
     storage.setItem("key1", "value1");
```

Comment:
```suggestion
    storage.setItem(key, "value1");
```

Use variable instead of duplicating string.